packages/application-components/src/components/portals-container/portals-container.tsx

```diff
@@ -36,49 +51,86 @@ const defaultProps: Pick<
   | 'containerSelectorToPreventScrollingOnOpen'
   | 'zIndex'
 > = {
-  offsetTop: '0',
-  offsetLeft: '0',
-  offsetLeftOnExpandedMenu: '0',
+  offsetTop: '0px',
+  offsetLeft: '0px',
+  offsetLeftOnExpandedMenu: '0px',
   containerSelectorToPreventScrollingOnOpen: 'main',
   zIndex: 10000,
 };
 
+const useObserverElementDimensions = (
```

Comment:
Will be used twice. Once for page and once for global nofications. Both wrapper divs will receive a ref so we can target them.